templates/home.html

```diff
@@ -133,7 +133,18 @@ <h1>What Slot</h1>
 					</tbody>
 				</table>
 			</div>
-
+			<style> 
+				#footer { 
+				  position: fixed; 
+				  padding: 13px 13px 0px 13px; 
+				  bottom: 0; 
+				  width: 100%; 
+												text-align: center;
+				  /* Height of the footer*/  
+				  height: 40px; 
+				  background: #ffffff; 
+				} 
+			  </style> 
```

Comment:
It is very tempting to write inline CSS. But trust me, it becomes very hard to find them afterwards.
Better keep the CSS in `static/main.css`.

Reply:
okay!!!!